hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -256,6 +256,8 @@ label.configuration.anonymous.download = Allow targets to download artifacts wit
 label.configuration.repository.autocleanup.action.notice = Warning: The actions are deleted from the repository and cannot be restored
 label.configuration.repository.multiassignments = Allow parallel execution of multiple distribution set assignments and rollouts
 label.configuration.repository.multiassignments.notice = Warning: Once this assignment behavior is active, it cannot be deactivated.
+label.configuration.repository.userconsent = Enable user consent flow
```

Comment:
Instead I would describe how the behaviour would change. Something like:

Request confirmation for actions before processing with download/install process